lib/dassets/config.rb

```diff
@@ -29,6 +30,24 @@ def set_base_url(value)
       @base_url = value
     end
 
+    def content_cache(cache = nil)
+      set_content_cache(cache) if !cache.nil?
+      @content_cache
+    end
+
+    def set_content_cache(cache)
+      @content_cache = cache
+    end
+
+    def fingerprint_cache(cache = nil)
+      set_fingerprint_cache(cache) if !cache.nil?
+      @fingerprint_cache
+    end
+
+    def set_fingerprint_cache(cache)
+      @fingerprint_cache = cache
+    end
```

Comment:
@jcredding I would remove these "set_*" methods if you don't have a use for them right now.  The `set_base_url` method is needed for testing (but that is it).